evm/src/executor/fork/backend.rs

```diff
@@ -704,10 +704,10 @@ mod tests {
     use foxar_common::get_http_provider;
     use foxar_config::Config;
     use std::{collections::BTreeSet, path::PathBuf, sync::Arc};
-    const ENDPOINT: &str = "https://mainnet.infura.io/v3/40bee2d557ed4b52908c3e62345a3d8b";
+    const ENDPOINT: &str = "http://127.0.0.1:8545/";
```

Comment:
Can't we put our archive node endpoint here? So the test works even if you are not running local node?

Reply:
Little bit lower we ignore 2 tests where we use this endpoint:
```#[ignore = "It doesn't work with our archive RPC node because it has https but not http. But with local node and http it works"]```
Function ```get_http_provider``` where we use endpoint do not work with https scheme